crates/extensions/c8y_config_manager/src/upload.rs

```diff
@@ -90,7 +90,7 @@ impl ConfigUploadManager {
                     self.upload_config_file(
                         Path::new(config_file_path.as_str()),
                         &config_upload_request.config_type,
-                        None,
+                        "main".into(),
```

Comment:
We should be passing the actual `device-id` here rather than this special key.

Reply:
done